src/main/java/org/energyweb/ddhub/Message.java

```diff
@@ -474,7 +474,7 @@ public Response searchInternal(@Valid @NotNull SearchInternalMessageDTO messageD
     @Authenticated
     public Response search(@Valid @NotNull SearchMessageDTO messageDTO)
             throws InterruptedException, TimeoutException, IOException, JetStreamApiException {
-        topicRepository.validateTopicIds(messageDTO.getTopicId(), true);
```

Comment:
I think this is also needed if topicId is not null or empty.

Reply:
that already update to topicRepository.validateTopicIds(messageDTO.getFqcnTopicList(), true);